doc/user/content/sql/create-table.md

```diff
@@ -9,49 +9,56 @@ menu:
     parent: 'commands'
 ---
 
-`CREATE TABLE` defines a table that is persisted in durable storage and can be
-written to, updated and seamlessly joined with other tables, views or sources.
+`CREATE TABLE` defines a table that is persisted in durable storage.
+
+In Materialize, you can create:
+- Read-write tables. With read-write tables, users can read ([`SELECT`]) and
+  write to the tables ([`INSERT`], [`UPDATE`], [`DELETE`]).
+
+- Read-only tables from [PostgreSQL sources (new
```

Comment:
heh ... I can't make up my mind between user-populated and read-only .. so, I've been going with both to catch both camps.  Eh ... probably when we do other sources, we'll be able to decide on one.